plugin/src/Plugin/GHCSpecter.hs

```diff
@@ -93,12 +94,40 @@ startSession opts env = do
   pid <- fromInteger . toInteger <$> getCurrentPid
   let modGraph = hsc_mod_graph env
       modGraphInfo = extractModuleGraphInfo modGraph
-      startedSession =
-        modGraphInfo `seq` SessionInfo pid (Just startTime) modGraphInfo False
+  sinfo <-
```

Comment:
nit: the name `ghcSessionInfo` was used in the later part of this function. be consistent.

Reply:
done.